mantidimaging/gui/windows/spectrum_viewer/model.py

```diff
@@ -317,6 +331,17 @@ def has_stack(self) -> bool:
         """
         return self._stack is not None
 
+    def has_normalise_stack(self) -> bool:
+        """
+        Check if data is available to export
+        @return: True if data is available to export and False otherwise
+        """
+        return self._normalise_stack is not None
```

Comment:
I can't see this being used anywhere

Reply:
Thanks for catching this! It was used at one point, but then I managed to get rid of the need for it after a few revisions so will drop it out now